nslsii/iocs/eps_two_state_ioc_sim.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+from caproto.server import pvproperty, PVGroup
+from caproto.server import template_arg_parser, run
+from caproto import ChannelType
+
+
+class EPSTwoStateIOC(PVGroup):
+    """
+    Simulates multiple-attempt issue for two-state device.
```

Comment:
```suggestion
    Simulates an EPS Two State device including multiple-attempt issue for two-state device.
    
    This IOC is used to simulate an EPS Two State Device, for testing or development purposes. Known EPS two state devices include Photon shutters, gate valves and Pneumatic actuators at NSLS-II. It simulates known issues with some of these including: A hardware error (when some attempts at sending the command fail and we need to 'kick' the device a few times to get it to actuate), A position status error (it sometimes does not reach the final state but remains 'between states') and an enable state change error (there is an 'enable' PV controlled by the control room that determines if the device can be operated or not). A parameter (described below) allows each of these error paths to be tested against
```

I prefer longer doc-strings as these are used in our documentation as well.